tools/bakeddata-scripts/src/main.rs

```diff
@@ -112,7 +112,16 @@ fn main() {
                 &std::fs::read_to_string(template.join(file))
                     .unwrap()
                     .replace("_component_", component)
-                    .replace("_version_", version),
+                    .replace("_version_", version)
+                    .replace("_cldr_tag_", DatagenProvider::LATEST_TESTED_CLDR_TAG)
+                    .replace(
+                        "_icuexport_tag_",
+                        DatagenProvider::LATEST_TESTED_ICUEXPORT_TAG,
+                    )
+                    .replace(
+                        "_segmenter_lstm_tag_",
+                        DatagenProvider::LATEST_TESTED_SEGMENTER_LSTM_TAG,
+                    ),
```

Comment:
Praise: This is, like... a super clean and simple way to do things. Good suggestion :+1: